src/FittedStar.cc

```diff
@@ -15,25 +15,23 @@ namespace jointcal {
 // cannot be in fittedstar.h, because of "crossed includes"
 FittedStar::FittedStar(const MeasuredStar &M) :
   BaseStar(M), mag(M.Mag()), emag(-1), col(0.), gen(-1), wmag(M.MagWeight()),
-  indexInMatrix(-1), measurementCount(0), refStar(nullptr),
+  indexInMatrix(-1), measurementCount(0), _refStar(nullptr),
   flux2(-1), fluxErr2(-1)
 {
   fluxErr = M.eflux;
 }
 
 
-void FittedStar::SetRefStar(const RefStar *R)
+void FittedStar::setRefStar(const RefStar *refStar)
 {
-  if (refStar != nullptr && (R)) // TODO: should we raise an Exception in this case?
+  if ((_refStar != nullptr) && (refStar != nullptr)) // TODO: should we raise an Exception in this case?
```

Comment:
I would say yes to this question. Just printing to `stderr` doesn't seem like a great idea. Don't we have a logging framework for non-fatal errors?